baselines/logging/src/v2/logging_service_v2_client.ts.baseline

```diff
@@ -0,0 +1,1799 @@
+        'folders/{folder}/exclusions/{exclusion}'
+      ),
+      folderLocationBucketPathTemplate: new this._gaxModule.PathTemplate(
+        'folders/{folder}/locations/{location}/buckets/{bucket}'
+      ),
+      folderLogPathTemplate: new this._gaxModule.PathTemplate(
+        'folders/{folder}/logs/{log}'
+      ),
+      folderSinkPathTemplate: new this._gaxModule.PathTemplate(
+        'folders/{folder}/sinks/{sink}'
+      ),
+      logMetricPathTemplate: new this._gaxModule.PathTemplate(
+        'projects/{project}/metrics/{metric}'
+      ),
+      organizationCmekSettingsPathTemplate: new this._gaxModule.PathTemplate(
+        'organizations/{organization}/cmekSettings'
+      ),
+      organizationExclusionPathTemplate: new this._gaxModule.PathTemplate(
+        'organizations/{organization}/exclusions/{exclusion}'
+      ),
+      organizationLocationBucketPathTemplate: new this._gaxModule.PathTemplate(
+        'organizations/{organization}/locations/{location}/buckets/{bucket}'
+      ),
+      organizationLogPathTemplate: new this._gaxModule.PathTemplate(
+        'organizations/{organization}/logs/{log}'
+      ),
+      organizationSinkPathTemplate: new this._gaxModule.PathTemplate(
+        'organizations/{organization}/sinks/{sink}'
+      ),
+      projectPathTemplate: new this._gaxModule.PathTemplate(
+        'projects/{project}'
+      ),
+      projectCmekSettingsPathTemplate: new this._gaxModule.PathTemplate(
+        'projects/{project}/cmekSettings'
+      ),
+      projectExclusionPathTemplate: new this._gaxModule.PathTemplate(
+        'projects/{project}/exclusions/{exclusion}'
+      ),
+      projectLocationBucketPathTemplate: new this._gaxModule.PathTemplate(
+        'projects/{project}/locations/{location}/buckets/{bucket}'
+      ),
+      projectLogPathTemplate: new this._gaxModule.PathTemplate(
+        'projects/{project}/logs/{log}'
+      ),
+      projectSinkPathTemplate: new this._gaxModule.PathTemplate(
+        'projects/{project}/sinks/{sink}'
+      ),
+    };
+
+    // Some of the methods on this service return "paged" results,
+    // (e.g. 50 results at a time, with tokens to get subsequent
+    // pages). Denote the keys used for pagination and results.
+    this._descriptors.page = {
+      listLogEntries:
+          new this._gaxModule.PageDescriptor('pageToken', 'nextPageToken', 'entries'),
+      listMonitoredResourceDescriptors:
+          new this._gaxModule.PageDescriptor('pageToken', 'nextPageToken', 'resourceDescriptors'),
+      listLogs:
+          new this._gaxModule.PageDescriptor('pageToken', 'nextPageToken', 'logNames')
+    };
+    // Some methods on this API support automatically batching
+    // requests; denote this.
+    this._descriptors.batching = {
+      WriteLogEntries: new this._gaxModule.BundleDescriptor(
+        'entries',
+        ['log_name','resource','labels'],null,
+        gax.createByteLengthFunction(
+          this._opts.fallback ?
```

Comment:
Just the simple `protoFilesRoot.lookup('google.logging.v2.LogEntry')` should work. Just like we do for LRO: 

```
    const waitResponse = protoFilesRoot.lookup(
      '.google.showcase.v1beta1.WaitResponse') as gax.protobuf.Type;
```

Reply:
Will need to have `protoFilesRoot` defined - again, just like for LRO:

```
{%- if (service.longRunning.length > 0) %}

    // This API contains "long-running operations", which return a
    // an Operation object that allows for tracking of the operation,
    // rather than holding a request open.
    const protoFilesRoot = opts.fallback?
      this._gaxModule.protobuf.Root.fromJSON(require("../../protos/protos.json")) :
      this._gaxModule.protobuf.loadSync(nodejsProtoPath);
```

We can change the condition to work for bundling methods as well.

Reply:
Trying, gax.createByteLengthFunction() takes params in this type:
```
const createByteLengthFunction: (message: {
    encode: (obj: {}) => {
        finish: () => {}[];
    };
}) => (obj: {}) => number
```

While argument of type 'Type' is not assignable to parameter of type '{ encode: (obj: {}) => { finish: () => {}[]; }; }'. It needs some type conversion I think

Reply:
done.